src/mui/detail/Show.js

```diff
@@ -50,6 +51,7 @@ export class Show extends Component {
             hasEdit,
             translate,
             version,
+            styles = defaultStyles,
```

Comment:
why use styles and not style ?

Reply:
because styles is an object that contain multiple styles